features/flags.json

```diff
@@ -158,17 +158,17 @@
       "lastEditedAt": "2022-05-02T17:12:41.421Z"
     },
     "mainnet": {
-      "enabled": true,
+      "enabled": false,
       "lastEditedBy": "ignatKorokov",
       "lastEditedAt": "2022-05-02T17:12:41.421Z"
     },
     "mainnet_STAGING": {
-      "enabled": true,
+      "enabled": false,
       "lastEditedBy": "ignatKorokov",
       "lastEditedAt": "2022-05-02T17:12:41.421Z"
     },
     "testnet_STAGING": {
-      "enabled": true,
+      "enabled": false,
```

Comment:
Not necessary to disable this, until it has a dedicated environment it should just be the same as `testnet`.